index.html

```diff
@@ -4,24 +4,72 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css"
+  />
+  <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
   <link rel="stylesheet" href="style.css">
   <title>Awsome books</title>
 </head>
 <body>
-
-  <h1>Awesome Books</h1>
+  <nav>
+    <h1> Awesome Books <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-book" viewBox="0 0 16 16">
+      <path d="M1 2.828c.885-.37 2.154-.769 3.388-.893 1.33-.134 2.458.063 3.112.752v9.746c-.935-.53-2.12-.603-3.213-.493-1.18.12-2.37.461-3.287.811V2.828zm7.5-.141c.654-.689 1.782-.886 3.112-.752 1.234.124 2.503.523 3.388.893v9.923c-.918-.35-2.107-.692-3.287-.81-1.094-.111-2.278-.039-3.213.492V2.687zM8 1.783C7.015.936 5.587.81 4.287.94c-1.514.153-3.042.672-3.994 1.105A.5.5 0 0 0 0 2.5v11a.5.5 0 0 0 .707.455c.882-.4 2.303-.881 3.68-1.02 1.409-.142 2.59.087 3.223.877a.5.5 0 0 0 .78 0c.633-.79 1.814-1.019 3.222-.877 1.378.139 2.8.62 3.681 1.02A.5.5 0 0 0 16 13.5v-11a.5.5 0 0 0-.293-.455c-.952-.433-2.48-.952-3.994-1.105C10.413.809 8.985.936 8 1.783z"/>
+    </svg>
+      </h1>
+      <ul>
+        <li><a id="list" href="#">List</a></li>
+        <li><a id="add" href="#">Add new</a></li>
+        <li><a id="contact" href="#">Contact</a></li>
+      </ul>
+  </nav>
   <div class="container">
 
   </div>
   <form action="#" method="post">
+    <h3 class="newBook">Add a new book</h3>
     <input type="text" name="title" id="title" placeholder = "Book title">
     <input type="text" name="author" id="author" placeholder = "Book author">
     <button id="submit" type="submit">Add</button>
   </form>
+  <section id="touch">
+    <h1>Conatct</h1>
+    <p>Do have any question or you just want to say "Hello"?
+      You can reach out to us!
+    </p>
+      <ul>
+        <li>Our email: Awesomebooks@microverse.com</li>
+        <li>Our phone : 00441789516</li>
+        <li>Our adress : Casabanca microverse</li>
+
+      </ul>
+  </section>
+
+  <footer>
```

Comment:
The footer does not follow the wireframe design provided. 

| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/183874869-492e2fc2-54e4-4eed-8cc6-67be52d71fdc.png) | ![image](https://user-images.githubusercontent.com/33268927/183874705-898cd92b-5ba4-4706-91dc-46e3423ba67d.png) |